clap_complete/tests/testsuite/dynamic.rs

```diff
@@ -122,6 +122,31 @@ help\tPrint this message or the help of the given subcommand(s)"
     );
 }
 
+#[test]
+fn suggest_hidden_possible_value() {
```

Comment:
This commit seems unrelated to the title of the PR.  Let's split this out into a separate PR and focus on getting the existing functionality merged

(and please do the two commit thing with tests)